src/devices/Hx711I2c/Hx711I2c.cs

```diff
@@ -0,0 +1,349 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Model;
+using System.Diagnostics;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Hx711
+{
+    /// <summary>
+    /// DFRobot KIT0176: I2C 1kg Weight Sensor Kit
+    /// </summary>
+    [Interface("DFRobot KIT0176: I2C 1kg Weight Sensor Kit")]
+    public class Hx711I2c : IDisposable
```

Comment:
So really looks like the same as the one I4ve pointed out. Just your is SPI, the other one is I2C. So would be great to reuse the naming, align some, etc.

Reply:
it's already mostly similar:
Offset - same
Tare - same
nano: Read; me: GetWeight
then there is couple of things they support and I don't and reverse

they have SampleAveraging I do support but do not expose directly

Reply:
also HX711 is GPIO and they seem to just use SPI for implementation. The module I have is only using HX711 as backend

Reply:
I added also SampleAveraging to match nano closer (also I found I needed that to get faster but less accurate sampling)